server.exs

```diff
@@ -1,3 +1,7 @@
+# iex -S mix dev
```

Comment:
Pay attention here, the file has been moved from the lib to the root of the library. It's not clear on github.